src/test/kotlin/no/nav/familie/ef/sak/mapper/StatsborgerskapMapperTest.kt

```diff
@@ -0,0 +1,34 @@
+package no.nav.familie.ef.sak.mapper
+
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.familie.ef.sak.integration.dto.pdl.Statsborgerskap
+import no.nav.familie.ef.sak.service.KodeverkService
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+
+internal class StatsborgerskapMapperTest {
+
+    private val kodeverkService: KodeverkService = mockk()
+    private val statsborgerskapMapper = StatsborgerskapMapper(kodeverkService)
+
+    @Test
+    internal fun `skal mappe NOR ISO3 landkode til Norge`() {
+        every { kodeverkService.hentLand(any(), any()) } returns "Norge"
```

Comment:
Hvis du sier at alt returnerer Norge, så tester du vel ikke at NOR mappes til Norge? `skal retunere resultat fra land for landkode fra kodeverk`?

Reply:
ff584e0